UndertaleModTool/Scripts/Builtin Scripts/RunSwitchAndXboxOnPC.csx

```diff
@@ -0,0 +1,173 @@
+// Sets up NXTALE to run well on PC.
+using System.Linq;
+using UndertaleModLib.Models;
+
+EnsureDataLoaded();
+
+if (Data.GeneralInfo.Name.Content != "NXTALE")
+{
+    ScriptError("This script can only be used with\nThe Nintendo Switch and Xbox One version of Undertale.", "Not NXTALE");
+    return;
+}
+
+bool isXbox = Data.Rooms.ByName("room_xbox_engagement") is not null;
+
+if (isXbox)
+{
+    // Fix initialization code not running
+    ReplaceTextInGML("gml_Object_obj_time_Create_0", @"
+if (os_type == os_xboxone)
+{
+    script_execute(SCR_GAMESTART, 0, 0, 0, 0, 0)", @"
+if (true)
+{
+    script_execute(SCR_GAMESTART, 0, 0, 0, 0, 0)");
+
+    var obj_time = Data.GameObjects.ByName("obj_time");
+    var otherEvents = obj_time.Events[(int)EventType.Other];
+    var gameStartEvent = otherEvents.FirstOrDefault(x => x.EventSubtype == (int)EventSubtypeOther.GameStart);
+    if (gameStartEvent is not null)
+        otherEvents.Remove(gameStartEvent);
+}
+
+// Disables interpolation. Only necessary for NX version.
+Data.GeneralInfo.Info &= ~(UndertaleGeneralInfo.InfoFlags.Interpolate);
+
+// Use new gamepad functions, because the compatibility ones are completely broken
+
+// use NX routine for joypad detection
+ReplaceTextInGML("gml_Object_obj_time_Create_0", @"
+if (global.osflavor >= 4)
+{
+    i = 0
+    while (i < gamepad_get_device_count())", @"
+if (true)
+{
+    i = 0
+    while (i < gamepad_get_device_count())");
```

Comment:
may just be me, but i'd prefer to use """ strings instead. (forgot how they're named)

Also is there a reason to include that much here? wouldnt just `if (global.osflavor >= 4)` > `if (true)` suffice?